src/compas_timber/fabrication/__init__.py

```diff
@@ -17,7 +18,7 @@
     "BTLxLap",
     "BTLxFrenchRidgeLap",
     "LButtFactory",
-    "TButtFactory",
+    # "TButtFactory",
```

Comment:
is it not imported anywhere? 
perhaps start by removing the registration of this factory in the original btlx package.